### PR TITLE
initial CLI ability

### PR DESCRIPTION
**Pull request by rogue-elephant**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
   "files": [
     "lib/**/*"
   ],
+  "bin": {
```
Comment (leemorton) [inline package.json]: Its this easy to tell npm that you offer a CLI?

Comment (rogue-elephant) [reply package.json]: pretty much

**Comment by rogue-elephant** [comment]
Initial CLI and did #17

**Comment by rogue-elephant** [comment]
closes #17

**Pull request merged**